src/main/resources/interface-specification.yml

```
@@ -567,7 +567,7 @@ paths:
             application/problem+json:
               schema:
                 $ref: '#/components/schemas/Problem'
-  /eservices/{eServiceId}/descriptors/{descriptorId}/documents:
+  /eservices/{eServiceId}/descriptors/{descriptorId}/documents/{documentId}/create:
```

a more RESTful approach is
```suggestion
  /eservices/{eServiceId}/descriptors/{descriptorId}/documents:
```
still using POST verb, passing the `documentId` in the payload